src/commands/targeting/disable.ts

```diff
@@ -4,6 +4,8 @@ import { Variation } from '../../api/schemas'
 import Base from '../base'
 import { renderTargetingTree } from '../../ui/targetingTree'
 import { getFeatureAndEnvironmentKeyFromArgs } from '../../utils/targeting'
+import { fetchAudiences } from '../../api/audiences'
+import { replaceAudienceIdsInFeatureConfigs } from '../../utils/audiences'
```

Comment:
looks like this import is no longer used?

Reply:
I'll clean up the imports